Src/Models/CBR.cpp

```diff
@@ -5,20 +5,26 @@
 
 #include <Misc/BinaryReader.h>
 
+#include "../Collision/CollisionMesh.h"
 #include "../Graphics/GraphicsResources.h"
 #include "../Utils/TextureUtil.h"
 
 const PGE::FilePath shaderPath = PGE::FilePath::fromStr("SCPCB/GFX/Shaders/RoomOpaque/");
 const PGE::FilePath shaderNormalPath = PGE::FilePath::fromStr("SCPCB/GFX/Shaders/RoomOpaqueNormalMap/");
-
 const PGE::String texturePath = "SCPCB/GFX/Map/Textures/";
 
+
 enum Lightmapped {
     No = 0,
     Fully = 1,
     Outdated = 2,
 };
 
+inline bool fileExists(PGE::String filename) {
+    struct stat buffer;
+    return (stat(filename.cstr(), &buffer) == 0);
```

Comment:
Avoid using POSIX.
FilePath::exists is a thing.

Reply:
Thankyou!
I was wondering where it was.
Fixed.